pyproject.toml

```diff
@@ -68,7 +68,7 @@ dependencies = [
     "click~=8.0.4",
     "requests>=2.31.0",
 ]
-requires-python = ">=3.9"
+requires-python = ">=3.10"
```

Comment:
nit: should update the `tools.black` block in this file to set the target version to py310